src/components/Dropdown/Dropdown.tsx

```diff
@@ -146,6 +146,20 @@ export class Dropdown extends BaseComponent<IDropdownProps, any> {
     }
   }
 
+  private _onRenderItem(item: IDropdownOption): JSX.Element {
```

Comment:
can you get rid of this function completely and just pull it from props in render (with the default being this._onRenderItem) like i showed in my previous comment